client/rpcserver/types.go

```diff
@@ -11,3 +13,31 @@ type VersionResult struct {
 	Minor uint32 `json:"minor"`
 	Patch uint32 `json:"patch"`
 }
+
+// ParseCmdArgs parses arguments to commands for rpcserver requests.
+func ParseCmdArgs(cmd string, args []interface{}) (interface{}, error) {
+	parse, exists := parsers[cmd]
+	if !exists {
+		return nil, fmt.Errorf("unknown command: %s", cmd)
+	}
+	return parse(args)
+}
+
+// parsers is a map of commands to parsing functions.
+var parsers = map[string](func([]interface{}) (interface{}, error)){
+	"help": func(args []interface{}) (interface{}, error) {
```

Comment:
Can we bring back the `parseHelpArgs` and `parseVersionArgs` functions and just reference them in the map?